forms/src/pages/SurveyCreationPage/components/form/AnswerSection/AnswersSection.scss

```diff
@@ -0,0 +1,20 @@
+.answers-section-answers-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.answers-section-answers-X-button {
+  margin-left: 1rem;
+  margin-top: 1rem;
+  border: 0;
+  background-color: white;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 25px;
+  cursor: pointer;
+}
+
+.answers-section-answers-X-button:active {
+  background-color: #9dc8fe;
```

Comment:
I suggest using mui's theme engine and specify standard accent colors to use across your app instead of hardcoding

https://mui.com/material-ui/customization/theming/